components/log-viewer-webui/client/src/ui/MainLayout.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from "react";
+
+import {
+    SearchOutlined,
+    UploadOutlined,
+} from "@ant-design/icons";
+import {
+    Layout,
+    Menu,
+    MenuProps,
+} from "antd";
+
+import "./MainLayout.css";
+
+
+const {Sider} = Layout;
```

Comment:
I believe Antd intentionally nested the component inside `Layout`. Would it be fine to use `<Layout.Sider/>` directly?

Reply:
forgot to post this before the approval -

i did some research and it seems that the destructing way is preferred because it makes the code more concise